webpack.config.js

```diff
@@ -163,10 +163,10 @@ function getWebpackConfig( { extensionName = '' } = {}, argv ) {
 					use: _.compact( [
 						MiniCssExtractPlugin.loader,
 						'css-loader',
-						extensionName && {
-							loader: 'namespace-css-loader',
-							options: `.${ extensionName }`, // Just the namespace class
-						},
+						// extensionName && {
```

Comment:
I haven't debugged where the `@keyframes missing name` error comes from, but usually that might happen when the namespace css loader would try to parse CSS that has a new line before the keyframe name, like this:

```
.selector-here{color:red}@keyframes
keyframeName
```

Reply:
in a separate PR I'd like to find these errors and fix them if they are errors

thanks for sharing - I hadn't any experience with it

Reply:
> thanks for sharing - I hadn't any experience with it

Well, that's just a wild guess.

> in a separate PR I'd like to find these errors and fix them if they are errors

I'd love to keep the namespacing in place - do you think we could fix these before landing this PR?